### ISSUE TITLE
Implement settings modal and remove device-switcher arrows from mute/camera buttons

### ISSUE THREAD

**Issue by jakewb-b** [vector-im/matrix-video-chat#131]

**Comment by SimonBrandner** [comment]
I don't think I understand why to remove the device-switcher arrows, they are a de-facto standard in most apps and changing devices through a dialog seems like a hassle...

**Comment by jakewb-b** [comment]
> 

I agree on web, however these are challenging to use on mobile, and as the mobile implementation is currently an adaptive/responsive web-app it is an extra level of complexity to have different controls on web versus mobile.

**Comment by SimonBrandner** [comment]
> > I don't think I understand why to remove the device-switcher arrows, they are a de-facto standard in most apps and changing devices through a dialog seems like a hassle...
> 
> I agree on web, however these are challenging to use on mobile, and as the mobile implementation is currently an adaptive/responsive web-app it is an extra level of complexity to have different controls on web versus mobile.

Ah, that makes sense! Thanks for a quick response!
